src/classes/GS_AdminSetup.cls

```diff
@@ -51,6 +53,72 @@ public with sharing class GS_AdminSetup {
         return checklistWrapers;
     }
 
+    /**
+     * @description Persist in database if an element is checked or not.
+     * For doing so we use the object GetStartedCompletionChecklistState__c;. 
+     * The object contains two fields: Item_Id and User. 
+     * Which means that a checkbox is selected if an element  
+     * it means the element is checked
+     * @param  itemId  Id the checklist item id
+     * @param  isChecked boolean element is checked of not
+     */
+    @AuraEnabled
+    public static void updateCheckItem(String itemId, Boolean isChecked){
+               
+        if (doesChecklistItemExists(itemId)) {
```

Comment:
Is it OK that this method checks for the existence of a record with this itemId but not specific to the current user (considering that the query in the if block **is** for the current UserId)?

Reply:
Yes, it is. It is validating that itemId exists in the CMT. It is independent of the User. If it is unchecked for the user, I wont find any record in the Object